packages/phaserx/src/createCamera.ts

```diff
@@ -53,6 +59,7 @@ export function createCamera(phaserCamera: Phaser.Cameras.Scene2D.Camera, option
   const wheelSub = wheelStream$
     .pipe(
       sampleTime(10),
+      filter((state) => !state.pinching),
```

Comment:
Thinking this should probably before the sample call so we don't drop events? Though it won't be too noticeable since we're sampling at 10ms.